vendors/ti/boards/cc3220_launchpad/ports/secure_sockets/iot_secure_sockets.c

```diff
@@ -902,6 +902,207 @@ int32_t SOCKETS_Send( Socket_t xSocket,
 }
 /*-----------------------------------------------------------*/
 
+#if ( configPLATFORM_SOCKET_UDP_SUPPORT == 1 )
+    int32_t SOCKETS_Bind( Socket_t xSocket,
+                          const SocketsSockaddr_t * pxAddress,
+                          Socklen_t xAddressLength )
+    {
+        SlSockAddrIn_t xAddress;
+        SSocketContextPtr_t pxSocketContext;
+        _i16 sTIRetCode;
+        int32_t lRetCode = SOCKETS_ERROR_NONE;
+        uint32_t ulSocketNumber = ( uint32_t ) xSocket;
+
+        /* Remove compiler warnings about unused parameters. */
+        ( void ) xAddressLength;
+
+        /* Ensure that the socket and address is valid. */
+        if( ( prvIsValidSocket( ulSocketNumber ) == pdTRUE ) || ( pxAddress != NULL ) )
+        {
+            /* Shortcut for easy access. */
+            pxSocketContext = &( xSockets[ ulSocketNumber ] );
+
+            /* Connect. */
+            if( lRetCode == SOCKETS_ERROR_NONE )
```

Comment:
`lRetCode` hasn't been modified since initialization on line 913.  Lines 926/927 & 946 can be removed.